packages/react/src/themes/teams/components/SplitButton/splitButtonVariables.ts

```diff
@@ -1,29 +1,30 @@
 import { SiteVariablesPrepared } from '../../../types'
-
 import { pxToRem } from '../../../../lib'
 
 export interface SplitButtonVariables {
   borderRadius: string
-  backgroundColorFocus: string
-  boxShadow: string
+  borderColorPrimary: string
   borderColor: string
-  colorFocus: string
-  primaryBackgroundColorFocus: string
-  primaryColorFocus: string
+  borderColorDisabled: string
+  borderWidthLeftOnly: string
```

Comment:
This variable name is specific to this implementation. If there is a need to have only a left border, then I'd put this directly in the styles file. If this needs to be a variable, it should be named something that only allows changing the left border, eg

`borderWidthLeftOnly: .1rem,`

and in styles files..
`borderWidth: 0 0 0 ${borderWidthLeftOnly},`

also, should we be consistent with using `pxToRem` or not in these situations?

Reply:
I switched to using borderWidth in SiteVariables and that's px instead of rem, so I'm not sure what we should be using